proto/cometbft/abci/v1/types.proto

```diff
@@ -307,7 +316,7 @@ message CheckTxResponse {
   reserved 9 to 11;
   reserved "sender", "priority", "mempool_error";
 
-  uint32 lane = 12;
+  Lane lane = 12;
```

Comment:
Why return the priority? The mempool already has all priorities from the handshake.